src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml

```diff
@@ -74,5 +74,11 @@
         <service id="Symfony\Component\Config\Resource\SelfCheckingResourceChecker">
             <tag name="config_cache.resource_checker" priority="-990" />
         </service>
+
+        <service id="Symfony\Component\HttpKernel\EventListener\ServiceResetListener">
+            <argument />
+            <argument type="collection" />
```

Comment:
can you add a small comment per abstract argument saying what should be injected?

Reply:
Done.